src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -160,11 +161,11 @@ public function wrapTable($table)
             $table = str_replace(' as ', ' ', $table);
         }
 
-        return $this->getSchemaPrefix() . $this->wrap($this->tablePrefix . $table, true);
+        return $this->getSchemaPrefix() . $this->tablePrefix . $table;
```

Comment:
Thanks for the ref. I tried quoting the table and it's not working. But I think I just to need to use upper case on it too. Will try and update this later.